src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/ExtendedInformation.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  useIntl,
+  FormattedMessage,
+} from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField,
+  Checkbox,
+  useCurrencyOptions,
+} from '@folio/stripes/components';
+
+import { PAYMENT_METHOD_OPTIONS } from '@folio/stripes-acq-components';
+
+import { AcceptedValuesField } from '../../../../../components';
+
+import { getFieldName } from '../../utils';
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+
```

Comment:
Remove this blank line between utils imports, please.

Reply:
done